farm/data_handler/data_silo.py

```diff
@@ -942,3 +942,104 @@ def _split_for_qa(documents, id_index, n_splits=5, shuffle=True, random_state=No
             current_train_set = np.hstack(np.delete(splits, idx, axis=0))
 
             yield current_train_set, current_test_set
+
+
+class DataSiloForNestedCrossVal(DataSiloForCrossVal):
```

Comment:
Now that I see the code there is a lot of duplicate code in this class and DataSiloForCrossVal.
Cant we merge both classes and parameterize it?

Reply:
I have done that. @Timoeller what do you think about this solution?